NitroxServer/ConsoleCommands/WhisperCommand.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.GameLogic;
+using NitroxModel.Packets;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Logger;
+
+namespace NitroxServer.ConsoleCommands
+{
+    class WhisperCommand : Command
+    {
+        private readonly PlayerManager playerManager;
+
+        public WhisperCommand(PlayerManager playerManager) : base("w", Perms.Player, Optional<string>.Of("w <PlayerName> <msg>"))
+        {
+            this.playerManager = playerManager;
+        }
+
+        public override void RunCommand(string[] args, Player player)
```

Comment:
Let's pass an Optional<Player> into these and we can do Optional<Player>.empty() if it is the server.  Then we don't have to check for the server id.